openmc/weight_windows.py

```diff
@@ -241,6 +260,9 @@ def to_xml_element(self):
         subelement = ET.SubElement(element, 'survival_ratio')
         subelement.text = str(self.survival_ratio)
 
+        subelement = ET.SubElement(element, 'max_lower_bound_ratio')
+        subelement.text = str(self.max_lower_bound_ratio)
```

Comment:
Since you're modifying XML, you should also update `docs/source/io_formats/settings.rst`